airflow/config_templates/airflow_local_settings.py

```diff
@@ -56,7 +56,10 @@
     'logging', 'DAG_PROCESSOR_MANAGER_LOG_LOCATION'
 )
 
-FILENAME_TEMPLATE: str = conf.get_mandatory_value('logging', 'LOG_FILENAME_TEMPLATE')
+# FILENAME_TEMPLATE only uses in Remote Logging Handlers since Airflow 2.3.3
+# All of these handlers inherited from FileTaskHandler and providing any value rather than None
+# would raise deprecation warning.
+FILENAME_TEMPLATE: Optional[str] = None
```

Comment:
I can't find any usage of this variable outside of `airflow/config_templates/airflow_local_settings.py`

Reply:
It shouldn’t be used anywhere else (should just read from `conf`). This can be removed.